packages/wrangler/src/deploy/index.ts

```diff
@@ -289,35 +290,29 @@ export async function deployHandler(
 		);
 	}
 
-	const experimentalAssetsBasePath = getExperimentalAssetsBasePath(
-		config,
-		args.experimentalAssets
-	);
-
-	if (
-		args.experimentalAssets &&
-		!existsSync(
-			path.resolve(experimentalAssetsBasePath, args.experimentalAssets)
-		)
-	) {
-		throw new UserError(
-			`The directory specified by the "--experimental-assets" command line argument does not exist:\n` +
-				`${path.resolve(experimentalAssetsBasePath, args.experimentalAssets)}`
+	const experimentalAssets = args.experimentalAssets
```

Comment:
this is merging args and config into one `ExperimentalAssets` shape for convenience. I also expect this will come in handy once we start to implement the dev/deploy behaviour, Same is done in `dev`